### ISSUE TITLE
have you tryed training with the full dataset?

### ISSUE THREAD

**Issue by kyoguan** [Kyubyong/tacotron#63]
<img width="941" alt="1gpu" src="https://user-images.githubusercontent.com/29282500/27847969-1a423f6e-6174-11e7-8933-71dd435fac19.png">

this result is training on single gpu. 

<img width="1007" alt="8gpu" src="https://user-images.githubusercontent.com/29282500/27847972-1d653f70-6174-11e7-9ef7-dceeba00a063.png">

this result is training on 8 gpu. (BTW: you need to set BN = None, or you would got a strange result , because the batch normal problem on multi gpu. )

it seems the adam optimizer problem.

**Comment by AzamRabiee** [comment]
![mean_loss](https://user-images.githubusercontent.com/29938304/27946144-4faf500c-632b-11e7-8306-5d79a3afa2c4.jpg)

**Comment by AzamRabiee** [comment]
My result on full dataset.
in epoch#40, I have better wav signal, as well.

**Comment by candlewill** [comment]
@AzamRabiee How is your synthesized  waves like at epoch 40? Could you share some of them?

**Comment by AzamRabiee** [comment]
[model_epoch_40_gs_40000_1.wav.zip](https://github.com/Kyubyong/tacotron/files/1130220/model_epoch_40_gs_40000_1.wav.zip)
Text is "abraham said i will swear"

**Comment by kyoguan** [comment]
I think you still using the sanity_check = True, this is only a very small dataset, I got the same result. can you try the sanity_check = False ?

**Comment by zuoxiang95** [comment]
In the code, if the sanity_check = True ,you use the data is only a single mini-batch, this batch repeats 1000 times:
texts, sound_files = texts[:hp.batch_size]*1000, sound_files[:hp.batch_size]*1000

**Comment by sniperwrb** [comment]
> I think you still using the sanity_check = True, this is only a very small dataset, I got the same result. can you try the sanity_check = False ?

I don't think it works.
I tried sanity_check=False, and used 40 epochs as well, because if I use 10000 it would cost me the whole summer :(
The result is a total mess. It is just repeating some certain phoneme that I cannot understand.

**Comment by tmulc18** [comment]
I did a non sanity check training (asynchronously on 384 cores) with  lr=.00015, norm=ins, loss=l1, min_len=10, max_len=100, and r=5.  Additionally, I used Luong Attention, gradient clipping by norm of 5, and binning.  Results are bad, but I ran out of cloud credits before I thought I was done training.
 
![image](https://user-images.githubusercontent.com/7132307/28611335-6b8892d4-7226-11e7-9004-53bc56fa014d.png)

After reading some of the comments here, I regret using any sort of normalization.  I will see if I can try a similar experiment again on a single gpu later this week, but it will be sure to take more time.


[samples%2Fmodel.ckpt-231691_25.wav.zip](https://github.com/Kyubyong/tacotron/files/1175964/samples.2Fmodel.ckpt-231691_25.wav.zip)

**Comment by sniperwrb** [comment]
@tmulc18 
I think normalization is somehow important.
When I try to eval.py with norm=None, the result is a mess (even for the pretrained model from the tacotron author)
When I try to eval.py with some normalizations, but the training did not have any, it would raise an error...

**Comment by jpdz** [comment]
Have you got good results? I trained the model based on the whole dataset. However, the loss is still, more than 1. And the results are not so good. Thanks a lot
